scale-ci-workload/Dockerfile

```diff
@@ -9,6 +9,15 @@ RUN mkdir -p $GOPATH/src/github.com/openshift-scale/perf-analyzer && \
     git clone https://www.github.com/openshift-scale/perf-analyzer $GOPATH/src/github.com/openshift-scale/perf-analyzer && \
     go get -d -v ./... && cd $GOPATH/src/github.com/openshift-scale/perf-analyzer && make
 
+# Build the conprof binary from source in a separate image
+
+WORKDIR $GOPATH/src/github.com/conprof/conprof
+
+RUN mkdir -p $GOPATH/src/github.com/conprof/conprof && \
+    git clone https://www.github.com/conprof/conprof $GOPATH/src/github.com/conprof/conprof && \
+    cd $GOPATH/src/github.com/conprof/conprof && GO111MODULE=on GOPROXY=https://proxy.golang.org go install -v
+# The next command is just to verify conprof binary was built into /go/bin, just a simple check
+# RUN ls /go/bin/
```

Comment:
We can remove the comments as the step which installs conprof will fail in case it's not able to build the binary. We need to fix it in case the image build is proceeding even when the conprof fails to build.

Reply:
agreed, removed